### PR TITLE
RM New metadb derived table po_acq_unit_ids

### PR DESCRIPTION
**Pull request by natalyapik**

### CODE REVIEW

**Review by jenmawe — CHANGES_REQUESTED** [review]
All queries:
**- [] Pull request has a complete description**, add description to PR
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented
- [x] Table and column names are in all-lowercase
**- [] Quotation marks are used only where necessary** au."name" -> au.name 

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Comment by nassibnassar** [comment]
- [ ] Pull request has a complete description
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/po_acq_unit_ids [hunk 1]
```diff
@@ -0,0 +1,25 @@
+-- These derived table extracts aquisition unit ids from purchase order table.
+DROP TABLE IF EXISTS po_acq_unit_ids;
+
+CREATE TABLE po_acq_unit_ids AS
+    SELECT
+        po.id AS po_id,
+        po.po_number AS po_number,
+        acq_unit_ids.jsonb #>> '{}' AS po_acquisition_unit_id,
```
Comment (stdombek) [inline sql_metadb/derived_tables/po_acq_unit_ids:8]: Please change to data type UUID.

File: sql_metadb/derived_tables/po_acq_unit_ids [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        au."name" AS po_acquisition_unit_name
```
Comment (stdombek) [inline sql_metadb/derived_tables/po_acq_unit_ids:9]: Please remove quotation marks.

**Review by stdombek — CHANGES_REQUESTED** [review]
- [ ] Pull request has a complete description
- [X] Query runs without errors
- [ ] Query output is correct
- [X] Query logic is clear and well documented
- [ ] Query is readable and properly indented
- [X] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary
- [X] Query begins with user documentation in comment lines
- [X] File name is listed in runlist.txt after dependencies
- [X] All columns have indexes
- [X] Table is vacuumed and analyzed

Very good. Please check the data type in line 8 and add a description to the PR. 

Query is not properly indented from line 5. The query should be one level to the left. But it's more because it's like that in other derived tables.

File: sql_metadb/derived_tables/po_acq_unit_ids (hunk 1 above)
Comment (stdombek) [reply sql_metadb/derived_tables/po_acq_unit_ids:8]: from 
`acq_unit_ids.jsonb #>> '{}' AS po_acquisition_unit_id`
to
`CAST(acq_unit_ids.jsonb #>> '{}' AS uuid) AS po_acquisition_unit_id`

**Review by adoerrer — CHANGES_REQUESTED** [review]
> * [ ]  Pull request has a complete description
> 
>     * [x]  Query runs without errors
> 
>     * [x]  Query output is correct
> 
>     * [x]  Query logic is clear and well documented
> 
>     * [x]  Query is readable and properly indented
> 
>     * [x]  Table and column names are in all-lowercase
> 
>     * [x]  Quotation marks are used only where necessary
> 
>     * [x]  Query begins with user documentation in comment lines
> 
>     * [x]  File name is listed in `runlist.txt` after dependencies
> 
>     * [x]  All columns have indexes
> 
>     * [x]  Table is vacuumed and analyzed


Query is good overall but the suffix `.sql` for the query is missing and a PR description is missing- You can just add it to the first box in the conversation.

**Review by stdombek — APPROVED** [review]
- [ ] Pull request has a complete description
- [X] Query runs without errors
- [X] Query output is correct
- [X] Query logic is clear and well documented
- [X] Query is readable and properly indented
- [X] Table and column names are in all-lowercase
- [X] Quotation marks are used only where necessary
- [X] Query begins with user documentation in comment lines
- [X] File name is listed in `runlist.txt` after dependencies
- [X] All columns have indexes
- [X] Table is vacuumed and analyzed

PR has no description. I think it can be ignored here.

**Review by adoerrer — APPROVED** [review]
I'd agree @stdombek since it might only an echo of PR title

**Review by stdombek — COMMENTED** [review]
In metadb, the IDs are changed to uuid unless it is specifically intended to do so within the app and would cause a chain reaction with the other reports using that ID.

**Review by stdombek — APPROVED** [review]
- [ ] Pull request has a complete description
- [X] Query runs without errors
- [X] Query output is correct
- [X] Query logic is clear and well documented
- [X] Query is readable and properly indented
- [X] Table and column names are in all-lowercase
- [X] Quotation marks are used only where necessary
- [X] Query begins with user documentation in comment lines
- [X] File name is listed in `runlist.txt` after dependencies
- [X] All columns have indexes
- [X] Table is vacuumed and analyzed

PR has no description. It can be ignored here.

**Pull request merged**